Soccar [UI]/src/nl/socnet/message/handler/JoinSessionMessageHandler.java

```diff
@@ -18,16 +19,28 @@
 
 /**
  * Handles JoinSession messages, after the currentPlayer has joined a session.
- * 
+ *
  * @author PTS34A
  */
 public final class JoinSessionMessageHandler extends MessageHandler<JoinSessionMessage> {
 
     @Override
     protected void handle(Connection connection, JoinSessionMessage message) throws Exception {
         Status status = message.getStatus();
-        if (status != Status.SUCCESS) {
-            return;
+        System.out.println(status.name());
+        switch (status) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a default case to this switch. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)